pooch/tests/data/tiny space data.txt

```diff
@@ -0,0 +1,2 @@
+# A tiny data file for test purposes only
```

Comment:
Is this file needed? Loading the registry doesn't look for the files mentioned there.

Reply:
Ah, I did not know that the files do not need to exist. That makes things easier, of course.